src/main/scala/nl/biopet/tools/findoverlapmatch/FindOverlapMatch.scala

```diff
@@ -119,4 +140,44 @@ object FindOverlapMatch extends ToolCommand[Args] {
     logger.info(s"multi $multiOverlap found")
     logger.info("Done")
   }
+
+  def descriptionText: String =
+    s"""
+       |$toolName looks for overlapping samples in a sample matrix.
+       |It compares samples and lists similar samples based on a cutoff point.
+       |It can also check if columns in a sample matrix match a certain regex.
```

Comment:
I think we should mention a link to vcfstats, this is the tool that will create such a matrix